java/src/test/java/ai/rapids/cudf/TableTest.java

```diff
@@ -327,6 +328,24 @@ void testReadJSONFile() {
     }
   }
 
+  @Test
+  void testReadSingleQuotesJSONFile() throws IOException {
+    Schema schema = Schema.builder()
+            .column(DType.STRING, "A")
+            .build();
+    JSONOptions opts = JSONOptions.builder()
+            .withLines(true)
+            .withNormalizeSingleQuotes(true)
+            .build();
+    try (Table expected = new Table.TestBuilder()
+            .column("TEST\"", "TESTER'")
+            .build();
+         MultiBufferDataSource source = sourceFrom(TEST_JSON_SINGLE_QUOTES_FILE);
+         Table table = Table.readJSON(schema, opts, source)) {
+            assertTablesAreEqual(expected, table);
+         }
```

Comment:
nit: inconsistent indent

```suggestion
      assertTablesAreEqual(expected, table);
    }
```